core/db/migrate/20210114182625_create_spree_payment_methods_stores.rb

```diff
@@ -0,0 +1,8 @@
+class CreateSpreePaymentMethodsStores < ActiveRecord::Migration[6.0]
+  def change
+    create_table :spree_payment_methods_stores, id: false do |t|
+      t.belongs_to :payment_method
+      t.belongs_to :store
+    end
```

Comment:
@MatthewKennedy we will also need a unique index on `[:payment_method_id, :store_id]`

Reply:
The migration creates the following Schema with these indexes:

```
  create_table "spree_payment_methods_stores", id: false, force: :cascade do |t|
    t.bigint "payment_method_id"
    t.bigint "store_id"
    t.index ["payment_method_id"], name: "index_spree_payment_methods_stores_on_payment_method_id"
    t.index ["store_id"], name: "index_spree_payment_methods_stores_on_store_id"
  end
```
(This is from SQLite)

Reply:
@MatthewKennedy gotcha, so we still need that unique index

Reply:
@damianlegawiec Will get the correct indexes sorted.

Reply:
New Schema with unique index:

```
  create_table "spree_payment_methods_stores", id: false, force: :cascade do |t|
    t.bigint "payment_method_id"
    t.bigint "store_id"
    t.index ["payment_method_id", "store_id"], name: "payment_mentod_id_store_id_unique_index", unique: true
    t.index ["payment_method_id"], name: "index_spree_payment_methods_stores_on_payment_method_id"
    t.index ["store_id"], name: "index_spree_payment_methods_stores_on_store_id"
  end
```